### DISCUSSION TITLE
Improve tooltip documentation

### DISCUSSION THREAD

**Discussion by hxtomer** [chakra-ui/chakra-ui#8838] [General] [answered]
### Description

Hi there!
When using tooltip with ref on custom component, it is mentioned in the docs that the function should accept ref, but it is also required to pass `{...props}` to the referenced inner component.

Thank you!

### Problem Statement/Justification

required extra params missing in the docs.

### Proposed Solution or API

Docs should be updated

**Comment by segunadebayo** [comment, marked as answer]
That tooltip note is from v2. In v3 the trigger uses `asChild`.

Put the ref and the extra props on the child you pass in. The current tooltip page shows this.
